README.md

```diff
@@ -118,6 +118,83 @@ Example for a subject like `Foo::Bar#baz` it will run all example groups with de
 `Foo::Bar#baz`, `Foo::Bar` and `Foo`. The order is important, so if mutant finds example groups in the
 current prefix level, these example groups *must* kill the mutation.
 
+Reading Mutation Output
+-----------------------
+
+Mutation output is grouped by selection groups. Each group contains three sections:
+
+1. An identifier for the current group.
+
+   **Format**:
+
+   ```text
+   [SELECTOR]:[SOURCE LOCATION]:[LINENO]
+   ```
+
+   **Example**:
+
+   ```text
+   Book#add_page:Book#add_page:/Users/johnbackus/Downloads/mutant-examples/lib/book.rb:18
+   ```
+
+2. A list of specs that mutant ran to try to kill mutations for the current group.
+
+   **Format**:
+
+   ```text
+   - [INTEGRATION]:0:[SPEC LOCATION]:[SPEC DESCRIPTION]
+   - [INTEGRATION]:1:[SPEC LOCATION]:[SPEC DESCRIPTION]
+   ```
+
+   **Example**:
+
+   ```text
+   - rspec:0:./spec/unit/book_spec.rb:9/Book#add_page should return self
+   - rspec:1:./spec/unit/book_spec.rb:13/Book#add_page should add page to book
+   ```
+
+3. A list of unkilled mutations diffed against the original unparser source
+
+   **Format**:
+
+   ```text
+   [MUTATION TYPE]:[SELECTOR]:[SOURCE LOCATION]:[SOURCE LINENO]:[IDENTIFIER]
+   [DIFF]
+   -----------------------
+   ```
+
+   - `[MUTATION TYPE]` will be one of the following:
+      - `evil` - a mutation of your source was not killed by your tests
+      - `neutral` your original source was injected and one or more tests failed
```

Comment:
I think we want to add an extra section explaining the implications of neutral errors? So we get a "unique" link to link to from the report?